seaclouds-on-byon.yaml

```diff
@@ -0,0 +1,63 @@
+name: SeaClouds platform
+
+location:
+  byon:
+    user: vagrant
+    privateKeyFile: ~/seaclouds-distribution/seaclouds_id_rsa
+    hosts:
+    - 192.168.100.10
+    - 192.168.100.11
+
+services:
+- serviceType: brooklyn.entity.basic.SameServerEntity
```

Comment:
Is there any way in Brooklyn to extract this common yaml as it's duplicated in https://github.com/SeaCloudsEU/seaclouds-distribution/blob/fix/simplify-installation/seaclouds.yaml ?